main.combodo-approval-extended.php

```diff
@@ -23,7 +23,16 @@
  * @author      Denis Flaven <denis.flaven@combodo.com>
  * @license     http://www.opensource.org/licenses/gpl-3.0.html LGPL
  */
-
+class ApprovalConfiguration
+{
+	const DEFAULT_TARGET = [
+		'UserRequest' =>   [
+		    'target_states' => ['new'],
+		    'bypass_profiles' => 'Service Manager',
+		    'reuse_previous_answers' => true,
```

Comment:
```suggestion
			'target_states' => ['new'],
			'bypass_profiles' => 'Service Manager',
			'reuse_previous_answers' => true,
```

Reply:
Please replace the spaces

Reply:
Why did you resolve the conversation? Do you mean it's intended to be partially spaces and partially tabs?